Dockerfile

```diff
@@ -6,6 +6,9 @@ ENV USE_OTEL=${USE_OTEL}
 # Set the USE_MSSQL env variable to true to enable SQL Server support
 ARG USE_MSSQL=true
 ENV USE_MSSQL=${USE_MSSQL}
+# Set default log config
+ARG LOG_CONFIG=/code/assets/production_log_config.json
```

Comment:
Why do we want to set the default to be production log config, not development?

Reply:
The Dockerfile by itself, its mostly used for customer installations, not for testing or locally development.  For example I'm guessing neither of you are running `docker build -t recordlinker .` on your laptops on a regular basis.

For development, we're using either the compose.yml (e.g. `docker compose -b run`) or just running the code natively on your machines (e.g. ./scripts/local_server.sh or uvicorn ...).

Since the Dockerfile on its own is just used for customers, it makes sense to set it appropriately.

However, I forgot that for our compose environments, using the non-json output is probably ideal!  I'll make that change in a moment.

Reply:
updated.  https://github.com/CDCgov/RecordLinker/pull/93/commits/faa8aefac2f49e3dd8c65a97be250272d51e56c9